source/mainnet/net/references/multi-sig.rst

```diff
@@ -0,0 +1,448 @@
+
+For example, to create a multi-sig transaction to send 1 CCD to an account on testnet using all local keys,
+run the following command:
+
+.. code-block:: console
+
+   $concordium-client transaction send --receiver 4bbdAUCDK2D6cUvUeprGr4FaSaHXKuYmYVjyCa4bXSCu3NUXzA --amount 1 --out ./transaction.json --energy 5000 --sender 4jxvYasaPncfmCFCLZCvuL5cZuvR5HAQezCHZH7ZA7AGsRYpix --grpc-port 20000 --grpc-ip node.testnet.concordium.com
+
+The multi-sig transaction is outputted into the ``transaction.json`` file and has the following format:
+
+.. code-block:: json
+   :force:
+
+   {
+      "energy": 5000,
+      "expiryTime": 1716995242,
+      "nonce": 46,
+      "payload": {
+         "amount": "1000000",
+         "toAddress": "4bbdAUCDK2D6cUvUeprGr4FaSaHXKuYmYVjyCa4bXSCu3NUXzA",
+         "transactionType": "transfer"
+      },
+      "signature": {
+         "0": {
+            "0": "3099534c5f32daf64dc40b7a0013979b9b74b167d259fc787a363ed2db7f1bcdafdcf06e166b2d915c7f29043186b3015a6064755bf3c3733bca2151b2b19c04"
+         }
+      },
+      "signer": "4jxvYasaPncfmCFCLZCvuL5cZuvR5HAQezCHZH7ZA7AGsRYpix",
+      "version": 1
+   }
+
+.. _sign-with-keyfile:
+
+If you want to sign the transaction with keys that are not in the local key directory, you need to provide
+the keys as a separate file with the ``--keys`` flag. Keys provided with a flag take precedence and no
+lookup of local keys in the key directory is performed.
+
+For example, to create a multi-sig transaction to send 1 CCD to an account on testnet using keys for signing
+from a file, run the following command:
+
+.. code-block:: console
+
+   $concordium-client transaction send --keys ./keypair.json --receiver 4bbdAUCDK2D6cUvUeprGr4FaSaHXKuYmYVjyCa4bXSCu3NUXzA --amount 1 --out ./transaction.json --energy 5000 --sender 4jxvYasaPncfmCFCLZCvuL5cZuvR5HAQezCHZH7ZA7AGsRYpix --grpc-port 20000 --grpc-ip node.testnet.concordium.com
+
+where ``keypair.json`` is a file of the following format:
+
+.. code-block:: json
+   :force:
+
+   {
+      "cidx": {
+         "kidx": {
+            "encryptedSignKey": {
+               "metadata": {
+                  "encryptionMethod": "AES-256",
+                  "iterations": ...,
+                  "salt": ...,
+                  "initializationVector": ...,
+                  "keyDerivationMethod": "PBKDF2WithHmacSHA256"
+               },
+               "cipherText": ...
+            },
+            "verifyKey": ...,
+            "schemeId": "Ed25519"
+         },
+         ...
+      },
+      ...
+   }
+
+
+If you want to create a multi-sig transaction with no signatures at all, you can use an
+empty ``keypair.json`` file of the following format:
+
+.. code-block:: json
+   :force:
+
+   {}
+
+.. note::
+
+   Choose an ``expiryTime`` for the transaction that takes into account the time it takes to gather all signatures
+   by the different entities. If the ``expiryTime`` has passed, the transaction is no longer valid.
```

Comment:
Should we not provide examples of how to choose an expiry time? Is this possible to do in the `transaction send` command? If so, we should add it there.